discover/deployment/articles/01-installing-search-engine/01-configuring-elasticsearch/03-logging.markdown

```diff
@@ -0,0 +1,124 @@
+how to enable the INFO log level for the embedded Elasticsearch server.
+
++$$$
+
+**Note:** These instructions show you how to adjust the embedded Elasticsearch
+server's logs. Logging for @product@'s search functionality is configurable via
+the 
+[Log Levels](/discover/portal/-/knowledge_base/7-1/server-administration) 
+screen at Control Panel &rarr; Configuration &rarr; Server Administration &rarr;
+Log Levels tool. Narrow down the list to include only the search classes and
+packages by searching for _com.liferay.portal.search_.
+
+To adjust logging for a remote Elasticsearch server, see 
+[Elastic's documentation](https://www.elastic.co/guide/en/elasticsearch/reference/6.1/logging.html).
+
+$$$
+
+Here's an example log message that _is_ displayed by default:
+
+    2018-09-13 16:49:24.442 WARN  [Elasticsearch initialization thread][EmbeddedElasticsearchConnection:315]
+
+Manage the log levels for the `EmbeddedElasticsearchConnection` in Server
+Administration.
+
+Here's an example log message that _isn't_ displayed by default:
+
+    [2018-09-05T17:25:30,107][WARN ][o.e.d.i.m.MapperService  ] [unmapped_type:string] should be replaced with [unmapped_type:keyword]
+
+To adjust logging for the `o.e.d.i.m.MapperService` and other Elasticsearch classes, 
+
+1.  Create a `/config` folder in `Liferay Home/data/elasticsearch6/`.
+
+2.  Create a `log4j2.properties` file in the new folder.
+
+3.  To enable INFO level logging, populate the `log4j2.properties` file with
+    these contents: 
+
+        appender.console.layout.pattern=[%d{ISO8601}][%-5p][%-25c{1.}] %marker%m%n
+        appender.console.layout.type=PatternLayout
+        appender.console.name=console
+        appender.console.type=Console
+        logger.action.level=info
+        logger.action.name=org.elasticsearch.action
+        rootLogger.appenderRef.console.ref=console
+        rootLogger.level=info
+        status=error
+
+    Read the Elasticsearch 
+    [logging documentation](https://www.elastic.co/guide/en/elasticsearch/reference/6.x/logging.html) 
+    for more information.
+
+4.  Restart the @product@ server, or just the embedded Elasticsearch.
+
+5.  To restart just the embedded Elasticsearch server, create a file called
+
+        com.liferay.portal.bundle.blacklist.internal.BundleBlacklistConfiguration.config 
+
+    in `Liferay Home/osgi/configs` directory and populate it with
+
+        blacklistBundleSymbolicNames="com.liferay.portal.search.elasticsearch6.impl"
+
+6.  Save the `.config` file and the bundle is stopped. Once you see the
+    `[STOPPED]` message in the logs, restart the bundle by commenting the line
+    out and re-saving the file:
+
+        #blacklistBundleSymbolicNames="com.liferay.portal.search.elasticsearch6.impl"
+
+Once this is accomplished, the embedded Elasticsearch server is properly set up
+to display Elasticsearch logs at the INFO level and above.
+
+<!-- Do we want to say a little more about what we're doing here? -->
```

Comment:
@rbohl 
can we mention that the below configs for slowlog and gc are for remote mode only